assets/scss/education.scss

```diff
@@ -0,0 +1,314 @@
+@import "./style.css";
```

Comment:
Why is the css file imported in scss? It is the one which is generated from the scss compiler.